Benchmarks/Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "benchmarks",
+    platforms: [
+        .macOS(.v14),
```

Comment:
Bumping the tools version to 5.9 seems like a good change to me, but we might want to leave the minimum OS deployment target here as macOS 13.3 since that's what the rest of our package is currently on. @iCharlesHu what do you think?

Reply:
I was thinking that maybe it was worthwhile to push deployment of the benchmarks too, to the baseline required for predicates (which don't work on 13.3 AFAIU?), but we can go back with #available checks instead if you prefer that.

Reply:
I agree. We should bump the tools version to 5.9

Reply:
Sounds good, in that case @hassila could we go back to the `#available` checks? That way it's easier for us to use this package for other pieces that don't require 14.0 in the future. Otherwise, this looks just about ready to me!

Reply:
Sure, no problem - will fix it up later today.

Reply:
Fixed in https://github.com/apple/swift-foundation/pull/291/commits/05a33a1fd0b85a80b871898ea09833093d0e648c